EventFilter/L1TRawToDigi/plugins/L1TValidationEventFilter.cc

```diff
@@ -95,22 +91,15 @@ bool
 L1TValidationEventFilter::filter(edm::Event& iEvent, const edm::EventSetup& iSetup)
 {
   using namespace edm;
-  
-  edm::Handle<FEDRawDataCollection> feds;
-  iEvent.getByToken(fedData_, feds);
 
-  if (!feds.isValid()) {
-    LogError("L1T") << "Cannot unpack: no FEDRawDataCollection found";
+  Handle<int> triggerCount;
+  iEvent.getByLabel(InputTag(src_.label(),"triggerCount"), triggerCount);
```

Comment:
It's usually preferable to let the configuration specify the whole InputTag, not only the module name with a hardcoded instance label.